{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "5fbb7186_5fd8925a",
        "filename": "vpd_tool_impl.cpp",
        "patchSetId": 22
      },
      "lineNbr": 550,
      "author": {
        "id": 1000522
      },
      "writtenOn": "2023-04-26T14:28:29Z",
      "side": 1,
      "message": "a) What are these magic numbers?\nb) How did you reach to these values?\nc) Why not adding timeout to the call API itself? \nd) Handle exception in case of failure required?\ne) Why this was needed in this case? A brief comment would help.",
      "range": {
        "startLine": 550,
        "startChar": 34,
        "endLine": 550,
        "endChar": 48
      },
      "revId": "8a99ce33a45faa3d58a7fe507b3075c2197dd857",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}